.github/actions/setup/action.yml

```
@@ -0,0 +1,48 @@
+name: Setup
+description: Common setup steps used by our workflows
+runs:
+  using: composite
+  steps:
+    - name: Setup pnpm
+      uses: pnpm/action-setup@v2
+      with:
+        version: 8.6.5
+
+    - name: Setup node 16.x
```

```suggestion
    - name: Setup node with nvmrc
```
Shouldn't have to remember to update this if nvmrc changes